core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -61,6 +66,7 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
 
   implicit val logging = new AkkaLogging(context.system.log)
 
+  val resourceManager = new AkkaClusterContainerResourceManager(context.system, instanceId, self, poolConfig)
```

Comment:
No, because those are still maintained strictly by the ContainerPool, regardless of ContainerResourceManager impl. These are not potentially "shared" resources, at this point at least.

Reply:
my confusion is with the naming of `resourceManager` which is a concern shared with multiple classes: ContainerPool, ContainerProxy, ContainerResourceManager, ContainerFactory.  I don't have a suggestion on how to solve this, but in a way, I wish this new class would have a name that would allow us to easily understand what it does, and how it stands up when compared to the other classes.